resources/views/variables/create.blade.php

```diff
@@ -73,11 +73,7 @@ class="fixed z-10 bottom-0 inset-x-0 px-4 pb-4 sm:inset-0 sm:flex sm:items-cente
                             <div class="absolute inset-0 bg-gray-500 opacity-75"></div>
                         </div>
 
-                        <div 
-                            @click.away="importOpen = false 
-                                        $wire.set('import', '')" 
-                            @keydown.escape.window="importOpen = false
-                                        $wire.set('import', '')"
+                        <div @click.away="importOpen = false" @keydown.escape.window="importOpen = false; $wire.set('import', '')"
```

Comment:
Re formatting to single line will break the behaviour. The conents of modal wont be cleared